File: Python/freefood.py
'''
 *
 *	Problem: "Free Food"
 *
'''

events = int(input())

datesList = []
while (events > 0):
	startDate, endDate = map(int, input().split(" "))
	for date in range(startDate, (endDate + 1)):
		datesList.append(date)
	events -= 1
	
datesSet = set(datesList)
print(len(datesSet))

File: Java/freefood.java
/*
 *	Problem: "Free Food"
 */

import java.util.*;

public class freefood {
	
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int events = sc.nextInt();
		
		Set<Integer> datesSet = new HashSet<>();
		while (events-- > 0) {
			int	startDate = sc.nextInt();
			int	endDate = sc.nextInt();
			
			for (int date = startDate; date <= endDate; date++) {
				datesSet.add(date);
			}
		}
		
		System.out.println(datesSet.size());
	}
	
}